source/messaging/channels-basics.rst

```diff
@@ -49,6 +49,15 @@ Notify teammates
 
 Let teammates know when they're needed by typing ``@username``.
 
+Link to Channels
+----------------
+
+Create a link to a channel by typing ``~`` followed by a channel name that you can enter or select.
+
+.. tip::
+  
+  Customers using Mattermost v6.1 or earlier can specify Public Channels only. Customers using Mattermost v6.2 or later can specify both Public and Private channels. 
```

Comment:
@cwarnermm aren't the docs normally updated to reference the latest version unless it's something major like the v6.0 layout. 

I really want us to avoid putting all these version statements in the docs for monthly product changes

Reply:
Fair point, @esethna. The original draft text wasn't adding value. I've reworked this update based on your feedback. I've simplified the version statement for customers on older Mattermost releases.